### PR TITLE
Support sofa service duplicate configurable

### PR DESCRIPTION
**Pull request by crazysaltfish**
Support sofa service duplicate configurable

### CODE REVIEW

**Review by HzjNeverStop — CHANGES_REQUESTED** [review]
please add unit test

File: sofa-boot-project/sofa-boot/src/main/resources/sofa-boot/log-codes.properties [hunk 1]
```diff
@@ -21,6 +21,7 @@
 01-00200=Can not found binding converter for binding type %s
 01-00201=Interface type is null. Interface type is required while publish a service
 01-00202=Argument delay must be a positive integer or zero
+01-00203=SofaService was already registered [%s]
```
Comment (HzjNeverStop) [inline sofa-boot-project/sofa-boot/src/main/resources/sofa-boot/log-codes.properties:24]: 异常信息可以和warn日志不一样，例如 Duplicate xxx ， please xxx

Comment (crazysaltfish) [reply sofa-boot-project/sofa-boot/src/main/resources/sofa-boot/log-codes.properties]: fixd

**Comment by crazysaltfish** [comment]
>

**Review by HzjNeverStop — APPROVED** [review]
LGTM

**Pull request merged**